packages/serve/src/index.ts

```diff
@@ -10,6 +10,12 @@ import parseArgs from './parseArgs';
  * @returns {Function} invokes the devServer API
  */
 export default function serve(...args: string[]): void {
+    try {
+        // eslint-disable-next-line node/no-extraneous-require
+        require('webpack-dev-server');
+    } catch (err) {
+        throw new Error(`You need to install 'webpack-dev-server' for running 'webpack serve'.\n${err}`);
+    }
```

Comment:
exitCode ?

Reply:
/cc @anshumanv

Reply:
exit code is missing in all tests, we'll have to change utils to export it, future PR

Reply:
I think we should use `process.exit(2)` here to avoid problems

Reply:
throwing err will exit the process, doing process.exit() after throw would be unreachable code

Reply:
So let's use `logger.error(); process.exit(2)`, because throw will be use `1` exit code, but `1` exit code only for webpack bundle problems